### PR TITLE
Fix: formatter printing path of files with todo

### PR DESCRIPTION
**Pull request by renatoi**
Fixed a bug where the formatter was printing the path of todo files when there were other errors in other files. The check if all messages are todos should be done for each result message array and not all results in total.

### CODE REVIEW

File: __tests__/__fixtures__/fixtures.ts [hunk 1]
```diff
@@ -22,4 +26,12 @@ export default {
     const eslintWithTodos = deepCopy(fixtures.eslintWithTodos);
     return tmp ? updatePaths(tmp, eslintWithTodos) : eslintWithTodos;
   },
+  eslintWithErrorsWarningsTodos(tmp?: string): ESLint.LintResult[] {
+    const eslintWithErrorsWarningsTodos = deepCopy(
+      fixtures.eslintWithErrorsWarningsTodos
+    );
+    return tmp
```
Comment (scalvert) [inline __tests__/__fixtures__/fixtures.ts:33]: Under what circumstance to we not want to provide a tmp dir to update the paths in the fixture with?

File: src/formatter.ts [hunk 2]
```diff
@@ -42,16 +41,16 @@ function formatResults(
       return;
     }
 
-    const areAllMessagesTodo =
-      counts.errorCount === 0 &&
-      counts.warningCount === 0 &&
-      counts.todoCount > 0;
+    const areAllMessagesTodo = messages.every(
+      (message) => message.severity === -1
```
Comment (scalvert) [inline src/formatter.ts:45]: It'd be great to have a const to represent the `-1` severity (obviously not on this line).

```
const TODO_SEVERITY = -1;
```

**Review by scalvert — CHANGES_REQUESTED** [review]

File: src/formatter.ts (hunk 2 above)
Comment (renatoi) [reply src/formatter.ts]: Good idea. Done. I created a const file that export this to other places too.

File: __tests__/__fixtures__/fixtures.ts (hunk 1 above)
Comment (renatoi) [reply __tests__/__fixtures__/fixtures.ts]: True, I removed it and made the default return.

**Review by scalvert — APPROVED** [review]

**Pull request merged**